src/redash_stmo/integrations/iodide/extension.py

```diff
@@ -31,6 +31,7 @@ def post(self, query_id):
         with open(self.TEMPLATE_PATH, "r") as template:
             source = template.read()
             context = {
+                "redash_url": settings.REDASH_URL,
```

Comment:
You can use `redash.handlers.authentication.base_href` instead, I bet.

Reply:
Fixed. Thanks for the suggestion!